src/apps/activity_assignments/domain/assignments.py

```diff
@@ -57,8 +57,8 @@ class ActivityAssignmentWithSubject(PublicModel):
     id: UUID
     activity_flow_id: UUID | None
     activity_id: UUID | None
-    respondent_subject: SubjectReadResponse
-    target_subject: SubjectReadResponse
+    respondent_subject: SubjectReadResponse | None
+    target_subject: SubjectReadResponse | None
```

Comment:
Just curious what's changed such that these can now become `None`. I recognize that in my proposed data structure, I suggested omitting the `targetSubject` from each assignment mainly in the interest of cutting down the response size, though I think there's little actual benefit there. But in your JSON sample you have both `targetSubject` and `respondentSubject` included, and with my cursory review, I _think_ it they'll continue to be present in the response (which also means less finicky FE work :+1:).

I was trying to verify if either is ever omitted in the response empirically, but I'm getting `[]` for all the `assignments` props of each activity/flow. I briefly tried digging into that to figure out why, but maybe I'll investigate that later.

Reply:
This was part of the original WIP commit on this branch, I just forgot to clean it up. Thanks for catching it. The intention came from your suggestion to omit the target subject, but I eventually decided against that to make things simpler.

Are you still always getting an empty list for the assignments?

Reply:
Thanks @sultanofcardio! No, I'm now getting assignments, I was just looking at a subject that actually didn't have any! 🤦🏻